resources/snakefiles/Snakefile-bin.smk

```diff
@@ -77,16 +77,21 @@ def get_contigs(sample, binning_df):
 
 
 include: "resources/snakefiles/qc.smk"
+include: "resources/snakefiles/assemble.smk"
 include: "resources/snakefiles/mapping.smk"
 
 rule map_all:
     input:
-        expand("output/binning/mapped_reads/{from_sample}.{to_sample}.sorted.bam",
+        expand("output/binning/bt2/mapped_reads/{from_sample}.{to_sample}.sorted.bam",
```

Comment:
Suggest changing this to:

```
expand("output/binning/{binner}/mapped_reads/{from_sample}.{to_sample}.sorted.bam",
                binner=config['binners'],
                from_sample=from_samples,
                to_sample=to_samples)
```

and then adding a:
```
binners = ['minimap2', 'bowtie2']
```
to the config file. That way, you don't *have* to bin with both every time you execute the pipeline.

Reply:
Great idea! Now you can just comment out one or the other.